lib/basket.js

```diff
@@ -160,8 +160,16 @@
 		head.appendChild( script );
 	};
 
+	var injectStyle = function( obj ) {
+		var style = document.createElement('style');
+		style.type = 'text/css';
+		style.text = obj.data;
+		head.appendChild( style );
+	};
+
 	var handlers = {
-		'default': injectScript
+		'default': injectScript,
+		'css': injectStyle
```

Comment:
Better use 'text/css' as key name instead 'css' cause obj.type must equal xhr.getResponseHeader('content-type'), which responsed as 'text/css', of course.